### DISCUSSION TITLE
Semaphore setup for slack integration

### DISCUSSION THREAD

**Discussion by ghost** [semaphoreui/semaphore#997] [General]
In the code for setup (`semaphore/cli/setup/setup.go`) there is a question for slack alerts. but I don't see it in my version (2.8.53). Is this in a version coming soon? We use slack for our internal communication, as well as announcements fro deployments. Having slack integration would be very useful. 

Thank you.

**Comment by fiftin** [comment]
It is available in v2.8.67. Docs will be soon.
